src/wp-includes/comment-template.php

```diff
@@ -24,7 +24,13 @@
 function get_comment_author( $comment_id = 0 ) {
 	$comment = get_comment( $comment_id );
 
-	$comment_id = ! empty( $comment->comment_ID ) ? $comment->comment_ID : (string) $comment_id;
```

Comment:
@hellofromtonya We have similar code in [get_comment_author_link()](https://github.com/WordPress/wordpress-develop/blob/trunk/src/wp-includes/comment-template.php#L227-L230) do we needs to update it?

Reply:
get_comment_author_link wasn't modified in the 6.6 cycle, so if it is also problematic it would be longer running. I think this would be a fine enhancement to make there, but isn't necessary right now

Reply:
I agree with @aaronjorbin to not include in 6.6.1. That said, @mukeshpanchal27 want to open a Trac ticket to improve it in the 6.7 cycle?

Reply:
@SergeyBiryukov Opened https://core.trac.wordpress.org/ticket/61715